internal/generator/ecc/template/marshal.go.tmpl

```diff
@@ -557,6 +557,48 @@ func (enc *Encoder) writeUint32(a uint32) error {
 
 {{ define "encode"}}
 
+func (enc *Encoder) writeG1Slice{{$.Raw}}(t []G1Affine) error {
```

Comment:
{{$.Raw}}

What is that in the func name ? :) 

another way to do this PR & ensure it's the same code is just in the type switch, case *[]X: encode(*v)

but yeah as long as there is no regression OK on my side 👍

Reply:
> another way to do this PR & ensure it's the same code is just in the type switch, case *[]X: encode(*v)
That's a good economic way to do it. I'll revert and do that.